packages/completer/src/handler.ts

```diff
@@ -92,6 +93,10 @@ export class CompletionHandler implements IDisposable {
     return this._isDisposed;
   }
 
+  set continuousHinting(value: boolean) {
```

Comment:
I would define a getter too. I found always weird an API that defines a setter but no getter. Alternatively you could convert this to a method `setContinuousHinting`.